proposals/lip-0061.md

```diff
@@ -418,8 +418,8 @@ If a validator node starts for the first time or loses all previously created si
 
 If for a validator `v`, after applying a block the value returned by the function `getBFTHeights().maxHeightPrecommitted` increases from `h1` to `h2`, then the following commit messages are created by the validator `v` and gossiped as described further below:
 
-* For every height `h` in `{h1+1, h1+2, ..., h2}` such that `existBFTParameters(h+1)` returns `True` (i.e., `h` is the height of a block authenticating a change of BFT parameters), validator `v` creates a commit message for the block `b` at height `h` in its chain if validator `v` was active at height `h`.
-* If `existBFTParameters(h2+1)` returns `False` (i.e., `h2` is not the height of a block authenticating a change of BFT parameters), `v` creates a commit message for the block `b` at height `h2` in its chain if validator `v` was active at height `h`.
+* For every height `h` in `{h1+1, h1+2, ..., h2}` such that `existBFTParameters(h+1)` returns `True` (i.e., `h` is the height of a block authenticating a change of BFT parameters) and `h >= MIN_CERTIFICATE_HEIGHT`, validator `v` creates a commit message for the block `b` at height `h` in its chain if validator `v` was active at height `h`.
```

Comment:
I guess that adding the condition `h >= MIN_CERTIFICATE_HEIGHT` is redundant, since 

`h >= h1+1 = getMaxRemovalHeight()+1 >=  MIN_CERTIFICATE_HEIGHT - 1 +1 = MIN_CERTIFICATE_HEIGHT `